cloudberry/neo/conf/application.conf

```diff
@@ -88,8 +88,9 @@ berry.query.gap = "1 day"
 
 #asterixdb.url = "http://localhost:19002/aql"
 asterixdb.url = "http://localhost:19002/query/service"
-
-#acceptable values: "AQL" or "SQLPP" or "sparksql"
-asterixdb.lang = SQLPP
+mysqldb.url = "jdbc:mysql://localhost:3306/sql?user=root&passwd=2048"
```

Comment:
I haven't found any solution to it.. maybe user have to change mysql's config file..